api/goodreads.py

```diff
@@ -12,31 +11,45 @@ def goodreads(update, context):
     if not query:
         text = "*Usage:* `/gr {BOOK_NAME}`\n"\
                "*Example:* `/gr stoner`"
+        parse_mode = 'Markdown'
     else:
         response = get(f'https://www.goodreads.com/search.xml?key={config["GOODREADS_API_KEY"]}&q={query}')
         root = ET.fromstring(response.content)
 
         for node in root.iter('work'):
             if node is not None:
-                title = node.find('best_book').find('title').text
-                author = node.find('best_book').find('author').find('name').text
-                # rating = node.find('average_rating').text
-                cover = _cover_URL(title)
-                stars = f"⭐ {node.find('average_rating').text}"
-
-                text = f"<b>{title}</b>\n{author}\n{stars}"\
-                       f"<a href='{cover}'>&#8205;</a>"
-
+                id = node.find('best_book').find('id').text
```

Comment:
Don't need the for loop this way
```suggestion
                id = root.findtext("search/results/work/best_book/id")
```
or
```suggestion
                id = node.findtext(".//work//best_book//id")
```